src/m365/spo/commands/contenttype/contenttype-list.ts

```diff
@@ -0,0 +1,74 @@
+import { Logger } from '../../../../cli';
+import { CommandOption } from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import SpoCommand from '../../../base/SpoCommand';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  webUrl: string;
+  category?: string;
+}
+
+class SpoContentTypeListCommand extends SpoCommand {
+  public get name(): string {
+    return commands.CONTENTTYPE_LIST;
+  }
+
+  public get description(): string {
+    return 'Lists all available content types in the specified site';
+  }
+
+  public commandAction(logger: Logger, args: CommandArgs, cb: (err?: any) => void): void {
+    let requestUrl: string = `${args.options.webUrl}/_api/web/ContentTypes`;
+
+    if (args.options.category){
+      requestUrl += `?$filter=Group eq '${args.options.category}'`;
```

Comment:
Since category is provided by the user, we should escape it using the `Utils.encodeQueryParameter` method before passing to the query

Reply:
done

Reply:
I used `encodeURIComponent` instead of `Utils.encodeQueryParameter` as I checked that it is more commonly used in the CLI code 🤔. I guess it's probably the same thing 🤔. Let me know if this is ok